.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: csharp
```

Comment:
@harman28 This will not work because Travis is not enabled on this repo at this moment in time. Can we please remove this and re-add it when travis works? 

We can create a separate PR for that.